app/Kconfig

```diff
@@ -95,6 +95,10 @@ config ZMK_IDLE_SLEEP_TIMEOUT
 
 endif
 
+config ZMK_EXT_POWER
+	bool "Enable support to control external power output"
+	default n
```

Comment:
I'm not sure defaulting `n` makes sense here. Without this, I think the nrfMicro won't automatically have VCC on, will it?

Reply:
Yep, it won't. Fixed it.